docs/remote/containers.md

```diff
@@ -228,6 +249,29 @@ Run **Remote-Containers: Attach to Running Container...** command from the Comma
 
 After a brief moment, a new window will appear and you'll be connected to the running container.
 
+### Attached container configuration files
+
+VS Code supports image-level configuration files to speed up setup when you repeatedly connect to a given container. Once attached, any time you open a folder or [install an extension](#managing-extensions), a local image specific configuration file will open in the editor with the appropriate updates that you can opt to save or edit. This json file support two properties:
+
+```json
+{
+    // Default path to open when attaching to a new container.
+    "workspaceFolder": "/path/to/code/in/container/here",
+
+    // An array of extension IDs that specify the extensions to
+    // install inside the container when you first attach to it.
+    "extensions": [
+        "dbaeumer.vscode-eslint"
+    ]
```

Comment:
Additionally "forwardPorts" can be set to an array of numbers, each representing a port on the container's side that will be forwarded automatically.

Reply:
Great - added to vnext.  Missed that one.